ledger/store/src/helpers/rocksdb/internal/id.rs

```diff
@@ -305,4 +306,7 @@ enum DataID {
     Test4,
     #[cfg(test)]
     Test5,
+
+    // Additional
+    DeploymentIDV2Map,
```

Comment:
nit: this new variant could also be above the `Test` ones, as those are only ever created in tests - never in the production database